### ISSUE TITLE
sst start exits abruptly

### ISSUE THREAD

**Issue by aarvay** [serverless-stack/serverless-stack#348]
The recent release breaks `sst start` for me. The node process just exits back to prompt with the following error. I think #347 might have introduced this.
```
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at truncateField (.../node_modules/@serverless-stack/cli/lib/object.js:39:12)
```

**Comment by fwang** [comment]
Oh good catch! Cutting a release right now with the fix.

**Comment by fwang** [comment]
This should be fixed in v0.14.1

**Comment by aarvay** [comment]
Great. Thanks for that quick fix 👍🏽
